### PR TITLE
Provide Reference guide for the Kanto System Metrics

### PR DESCRIPTION
**Pull request by gboyvalenkov-bosch**
[#49] Provide Reference guide for the Kanto System Metrics

- reference guide created

Signed-off-by: Georgi Boyvalenkov <Georgi.Boyvalenkov@bosch.io>

### CODE REVIEW

File: web/site/content/docs/references/system-metrics-config.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: "System metrics configuration"
+type: docs
+description: >
+  Customize the system metrics.
```
Comment (e-grigorov) [inline web/site/content/docs/references/system-metrics-config.md:5]: Here, we have a focus on the most important functionality, not on the component name. Look around and check the rest of the descriptions.

File: web/site/content/docs/references/system-metrics-config.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+weight: 5
+---
+
+### Properties
+
+To control all aspects of the system metrics behavior.
+
+| Property | Type | Default | Description |
+| - | - | - | - |
+| frequency | string | 0s | Initial frequency of publishing system data to cloud as a sequence of decimal numbers, each with optional fraction and a unit suffix, such as: 300ms, 1.5h, 10m30s, etc., time units are: ns, us (or µs), ms, s, m, h |
```
Comment (e-grigorov) [inline web/site/content/docs/references/system-metrics-config.md:15]: - could you please check the default value? I am not sure that there is such.
- "to cloud" should be removed. Eclipse Hono/Ditto can be deployed in different environments: cloud, on-premise etc.

File: web/site/content/docs/references/system-metrics-config.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+title: "System metrics configuration"
+type: docs
+description: >
+  Customize the system metrics.
+weight: 5
+---
+
+### Properties
+
+To control all aspects of the system metrics behavior.
```
Comment (e-grigorov) [inline web/site/content/docs/references/system-metrics-config.md:11]: Focus on the functionality, not the component name.

File: web/site/content/docs/references/system-metrics-config.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+| **Logging** | | | |
+| logFile | string | log/system-metrics.log | Path to the file where log messages are written |
+| logLevel | string | INFO | All log messages at this or higher level will be logged, the log levels in descending order are: ERROR, WARN, INFO, DEBUG and TRACE |
+| logFileCount | int | 5 | Log file maximum rotations count |
+| logFileMaxAge | int | 28 | Log file rotations maximum age in days, use 0 to not remove old log files |
+| logFileSize | int | 2 | Log file size in MB before it gets rotated |
+
+### Example
+
+The minimal required configuration that runs system metrics.
```
Comment (e-grigorov) [inline web/site/content/docs/references/system-metrics-config.md:29]: > runs system metrics

Run is not the main reason. The minimum configuration to activate.....

File: web/site/content/docs/references/system-metrics-config.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+```json
+{
+"frequency": "60s"
```
Comment (e-grigorov) [inline web/site/content/docs/references/system-metrics-config.md:33]: Could you please apply a formatter. See the rest of the examples.

**Review by e-grigorov — CHANGES_REQUESTED** [review]

File: web/site/content/docs/references/system-metrics-config.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+### Template
+
+The configuration can be further adjusted according to the use case.
+The following template illustrates all possible properties with their default values.
+
+```json
+{
+    "frequency" : "0s"
```
Comment (e-grigorov) [inline web/site/content/docs/references/system-metrics-config.md:45]: Finally, is 0s a default value or not? It was removed from the table.

**Review by e-grigorov — CHANGES_REQUESTED** [review]

File: web/site/content/docs/references/system-metrics-config.md (hunk 6 above)
Comment (gboyvalenkov-bosch) [reply web/site/content/docs/references/system-metrics-config.md]: Fixed

File: web/site/content/docs/references/system-metrics-config.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+| **Logging** | | | |
+| logFile | string | log/system-metrics.log | Path to the file where log messages are written |
+| logLevel | string | INFO | All log messages at this or higher level will be logged, the log levels in descending order are: ERROR, WARN, INFO, DEBUG and TRACE |
+| logFileCount | int | 5 | Log file maximum rotations count |
+| logFileMaxAge | int | 28 | Log file rotations maximum age in days, use 0 to not remove old log files |
+| logFileSize | int | 2 | Log file size in MB before it gets rotated |
+
+### Example
+
+The minimal required configuration that enables reporting of system metrics.
```
Comment (e-grigorov) [inline web/site/content/docs/references/system-metrics-config.md:29]: One final suggestion, the reporting can be enabled with a command or with this configuration property. The configuration property is more like auto-reporting. This sentence can be extend to: ...enables the auto-reporting...

**Review by e-grigorov — CHANGES_REQUESTED** [review]

File: web/site/content/docs/references/system-metrics-config.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+weight: 5
+---
+
+### Properties
+
+To control all aspects of the system metrics behavior.
+
+| Property | Type | Default | Description |
+| - | - | - | - |
+| frequency | string | | Initial frequency of reporting system metrics as a sequence of decimal numbers, each with optional fraction and a unit suffix, such as: 300ms, 1.5h, 10m30s, etc., time units are: ns, us (or µs), ms, s, m, h |
```
Comment (konstantina-gramatova) [inline web/site/content/docs/references/system-metrics-config.md:15]: Could be simplified to:
Initial system metrics reporting frequency as a sequence of decimal numbers, each with an optional fraction and a unit...............

**Review by konstantina-gramatova — CHANGES_REQUESTED** [review]

**Review by e-grigorov — APPROVED** [review]

**Review by konstantina-gramatova — APPROVED** [review]

**Pull request merged**